apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -881,14 +882,13 @@ const migrateConversationsIfRequiredEpic: AppEpic = (action$, state$) => {
           return EMPTY;
         }
 
-        const sortedConversations = notMigratedConversations.sort((a, b) => {
-          if (!a.lastActivityDate) return 1;
-          if (!b.lastActivityDate) return -1;
-
-          return a.lastActivityDate - b.lastActivityDate;
-        });
+        const sortedConversations = orderBy(
+          notMigratedConversations,
+          [(c) => c.lastActivityDate === null, (c) => c.lastActivityDate],
```

Comment:
I would prefer something like
```javascript
[
    ...withoutDate,
    ...sortBy(withDate, 'lastActivityDate')
]
```
because it's more readable and obvious